### PR TITLE
Update MSFT_xDSCWebService to support registrationKeys

### PR DESCRIPTION
**Pull request by rayhayes**
Update MSFT_xDSCWebService to support registrationKeys

### CODE REVIEW

**Comment by rayhayes** [comment]
I'm putting it in the correct branch this time.

File: xPSDesiredStateConfiguration.psd1 [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @{
 # Version number of this module.
-ModuleVersion = '3.1.3.4'
+ModuleVersion = '3.2.0.0'
```
Comment (TravisEz13) [inline xPSDesiredStateConfiguration.psd1]: Don't change version, this will be done when we merge to master

File: README.md [hunk 2]
```diff
@@ -120,6 +120,15 @@ Domain members may be specified using domain\name or Universal Principal Name (U
 
 ## Versions
 
+### 3.2.0.0
```
Comment (TravisEz13) [inline README.md]: unrelease

**Comment by TravisEz13** [comment]
and this is into your repo, not powershell's

**Pull request closed without merging**